docs/pages/machine-id/access-guides/ssh.mdx

```diff
@@ -3,4 +3,155 @@ title: Machine ID with Server Access
 description: How to use Machine ID to access servers via SSH
 ---
 
-TODO
\ No newline at end of file
+(!docs/pages/includes/machine-id/v2-config-warning.mdx!)
+
+Teleport Server Access protects and controls SSH access to servers. Machine ID
+can be used to grant machines secure, short-lived access to these servers.
+
+In this guide, you will configure `tbot` to produce credentials that can be
+used to access a Linux server enrolled in Teleport Server Access. The guide
+will cover access using Teleports CLI `tsh` as well as the OpenSSH SSH client.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- If you have not already connected your server to Teleport, follow
+  the [Server Access Getting Started Guide](../../application-access/getting-started.mdx).
+
+- (!docs/pages/includes/tctl.mdx!)
+
+- `tbot` must already be installed and configured on the machine that will
+  connect to Linux hosts with SSH. For more information, see the
+  [platform guides](../platform-guides.mdx).
+
+## Step 1/3. Configure RBAC
+
+First, Teleport must be configured to allow the credentials produced by the bot
+to connect to the SSH servers. In this example, access will be granted to all
```

Comment:
```suggestion
to connect to SSH servers in your infrastructure. In this example, access will be granted to all
```
Qualifying this since it's the first time we mention "SSH servers".